exporter/datadogexporter/config.go

```diff
@@ -0,0 +1,117 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datadogexporter
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// DogStatsDConfig defines the DogStatsd related configuration
+type DogStatsDConfig struct {
+	// Endpoint is the DogStatsD address.
+	// The default value is 127.0.0.1:8125
+	// A Unix address is supported
+	Endpoint string `mapstructure:"endpoint"`
+
+	// Telemetry states whether to send metrics
+	Telemetry bool `mapstructure:"telemetry"`
```

Comment:
Is this like internal metrics about statsd or all metrics in general?  I would clarify the description of this.